src/overlays/actors/ovl_En_Arrow/z_en_arrow.h

```diff
@@ -40,5 +40,4 @@ typedef enum {
     /*  9  */ ARROW_SEED,
     /*  10 */ ARROW_NUT
 } ArrowType;
-
```

Comment:
I would leave that empty line in for consistency with all other headers (you may have not intended to commit it lol)

Reply:
No I didn't, vscode being nosy again